manifests/status.pp

```diff
@@ -73,23 +73,12 @@
     before  => Anchor['mysql::server::end'],
   }
 
-  augeas { 'mysqlchk':
-    context => '/files/etc/services',
-    changes => [
-      "rm /files/etc/services/service-name[port = '${port}']",
-      "set /files/etc/services/service-name[port = '${port}']/port ${port}",
-      "set /files/etc/services/service-name[port = '${port}'] mysqlchk",
-      "set /files/etc/services/service-name[port = '${port}']/protocol tcp",
-    ],
-    onlyif  => "match service-name[. = 'mysqlchk'] size == 0",
-    before  => Anchor['mysql::server::end'],
-  }
-
   xinetd::service { 'mysqlchk':
     server                  => '/usr/local/bin/clustercheck',
     port                    => $port,
     user                    => 'clustercheck',
     flags                   => 'REUSE',
+    service_type            => 'UNLISTED',
```

Comment:
Does this work on all plattforms?

Reply:
I can't actually test on any platform but CentOS 7, but a quick check of the xinetd.conf man page on Ubuntu 12.04 and RHEL5 mentions this as an option, so at least any reasonably recent version of xinetd should support this.